scripts/gen_dynamic_openapi.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+set -e
+
+# Generate an OpenAPI document for all backends.
+#
+# Assumptions:
+#
+#   1. Vault has been checked out at an appropriate version and built
+#   2. vault executable is in your path
+#   3. Vault isn't already running
+
+echo "Starting Vault..."
+if pgrep -x "vault" > /dev/null
+then
+    echo "Vault is already running. Aborting."
+    exit 1
+fi
+
+vault server -dev -dev-root-token-id=root &
+sleep 2
+VAULT_PID=$!
+
+echo "Mounting all builtin backends..."
+
+# Read auth backends
+flag=false
+re='".*"'
```

Comment:
Wouldn't this regex match anything? We probably want to have a more narrow regex here.

Reply:
> I definitely like this direction 👍 I can help with tightening up the shell script & the regex.

Cool! I'll publish this one and start addressing your comments!

Reply:
This matches everything between quotes, which seemed specific enough to pick up what we need. But let me know improving regex. It could be narrower to include `:` and the key value pattern. But it seemed okay since we're just regexing one line. Let me know your thoughts though